docs/changelog.rst

```diff
@@ -4,6 +4,18 @@ Release Notes
 
 .. towncrier release notes start
 
+3.4.1 (2023-09-13)
```

Comment:
You can remove this change, it will be auto-generated by Towncrier. The file in the `news/` directory is sufficient.